### PR TITLE
Fixes and changes to bootstrap accessibility plugin

### PR DESCRIPTION
**Pull request by cpsievert**
This fixes the same bug as https://github.com/rstudio/shiny/pull/3256 (for `theme = bslib::bs_theme(version = 3)`) and also changes `$mouse-outline-style` to `none` so that you must opt-in with something like this:

https://github.com/paypal/bootstrap-accessibility-plugin has several issues:

(1) [It assumes that the page is already loaded before it runs](https://github.com/paypal/bootstrap-accessibility-plugin/blob/fbbf870eafc1ee5d4547fabbeea71778ddbe2166/src/js/tab.js#L4-L12), so the aria labels aren't added on page load to something like this:

```{r}
library(shiny)
ui <- fluidPage(
  theme = bslib::bs_theme(version = 3),
  tabsetPanel(
    tabPanel("A", "a"),
    tabPanel("B", "b")
  )
)
server <- function(input, output) {}
shinyApp(ui, server)
```

(2) It always provides outlines for mouse interactions, which is quite a deviation in visual style, and probably annoying most users, so we've made these mouse outlines an opt-in feature via a Sass variable called `$mouse-outline-style`:

```r
library(shiny)
ui <- fluidPage(
  theme = bslib::bs_theme(version = 3, "mouse-outline-style" = "auto"),
  tabsetPanel(
    tabPanel("A", "a"),
    tabPanel("B", "b")
  )
)
server <- function(input, output) {}
shinyApp(ui, server)
```

(3) As a consequence of opt-ing out of mouse-based outlines, keyboard navigation suffers on Chrome, so we've decided to enable `:focus-visible` outlines by default, so keyboard navigation is supported on browsers that support it.

```r
library(shiny)
ui <- fluidPage(
  theme = bslib::bs_theme(version = 3, "keyboard-outline-color" = "red"),
  tabsetPanel(
    tabPanel("A", "a"),
    tabPanel("B", "b")
  )
)
server <- function(input, output) {}
shinyApp(ui, server)
```

### CODE REVIEW

File: tools/yarn_install.R [hunk 1]
```diff
@@ -336,6 +337,44 @@ for (patch in list.files(patch_dir, full.names = TRUE)) {
...
 }
 
+# ----------------------------------------------------------------------
+# Apply minification to patched files
+# ----------------------------------------------------------------------
+
+withr::with_dir("inst", {
+  local({
+    # install all deps
+    system("yarn install")
```
Comment (wch) [inline tools/yarn_install.R:347]: Why no `--production` here if it's added above?

File: tools/yarn_install.R [hunk 2]
```diff
@@ -11,7 +11,8 @@ if (Sys.which("yarn") == "") {
   stop("The yarn CLI must be installed and in your PATH")
 }
 
-withr::with_dir("inst", system("yarn install"))
+# only install the direct deps
+withr::with_dir("inst", system("yarn install --production"))
```
Comment (wch) [inline tools/yarn_install.R:15]: Are we sure that this is the right way to go?

File: inst/lib/bs-a11y-p/src/sass/partials/_mixins.scss [hunk 3]
```diff
@@ -1,3 +1,6 @@
 @mixin outline($size) {
```
Comment (wch) [inline inst/lib/bs-a11y-p/src/sass/partials/_mixins.scss:1]: Can you add a comment that says that `outline` typically isn't visible?

Comment (wch) [reply inst/lib/bs-a11y-p/src/sass/partials/_mixins.scss:1]: Or possibly rename `outline` to something that makes more sense.

**Review by wch — APPROVED** [review]

File: tools/yarn_install.R (hunk 2 above)
Comment (cpsievert) [reply tools/yarn_install.R:15]: This makes it so we can use `devDependencies` without having to clean-up parcel

File: tools/yarn_install.R (hunk 1 above)
Comment (cpsievert) [reply tools/yarn_install.R:347]: So that `yarn parcel` is available (it isn't with `--production`)

**Review by schloerke — APPROVED** [review]

**Pull request merged**